incubator/exporter-stdout/src/main/java/io/aklivity/zilla/runtime/exporter/stdout/internal/stream/SchemaRegistryEventHandler.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.exporter.stdout.internal.stream;
+
+import java.io.PrintStream;
+import java.util.function.LongFunction;
+
+import org.agrona.DirectBuffer;
+
+import io.aklivity.zilla.runtime.exporter.stdout.internal.types.event.SchemaRegistryEventFW;
+import io.aklivity.zilla.runtime.exporter.stdout.internal.types.event.SchemaRegistryRemoteAccessRejectedEventFW;
+
+public class SchemaRegistryEventHandler extends EventHandler
+{
+    private static final String SCHEMA_REGISTRY_REMOTE_ACCESS_REJECTED =
```

Comment:
```suggestion
    private static final String REMOTE_ACCESS_REJECTED =
```
We are already in the context of the `SchemaRegistryEventHandler`.

Reply:
OK